Src/Fido2/AttestationFormat/AndroidKey.cs

```diff
@@ -134,21 +134,20 @@ public override (AttestationType, X509Certificate2[]) Verify()
     {
         // 1. Verify that attStmt is valid CBOR conforming to the syntax defined above and perform CBOR decoding on it to extract the contained fields
         // (handled in base class)
-        if (attStmt.Count is 0)
-            throw new Fido2VerificationException(Fido2ErrorCode.InvalidAttestation, "Attestation format android-key must have attestation statement");
+        if (_attStmt.Count is 0)
+            throw new Fido2VerificationException(Fido2ErrorCode.InvalidAttestation, Fido2ErrorMessages.MissingAndroidKeyAttestationStatement);
 
-        if (!(Sig is CborByteString { Length: > 0 }))
-            throw new Fido2VerificationException(Fido2ErrorCode.InvalidAttestation, "Invalid android-key attestation signature");
+        if (!TryGetSig(out byte[]? sig))
```

Comment:
It's unfortunate that this performance boost is a little less readable. Never been a fan of  `out`, but I think the trade off is OK, just unfortunate.